src/module.c

```diff
@@ -97,7 +97,25 @@ static void RG_AfterForkChild() {
 	process_is_child = true;
 }
 
-static void RegisterForkHooks() {
+static int _RenameGraphContext(RedisModuleCtx *ctx, int type, const char *event,
```

Comment:
Rename to `RenameGraphHandler`

Reply:
Also please add some comment explaining why and how

Reply:
done